qiskit/circuit/__init__.py

```diff
@@ -806,14 +806,8 @@
     q_1: ───────────┤ X ├───┤ X ├───┤ X ├     q_1: ───┤ X ├───
                     └───┘   └───┘   └───┘             └───┘
 
-Performing these optimizations are part of the transpiler, but the tools to investigate commutations
-are available in the :class:`CommutationChecker`.
-
-.. autosummary::
-   :toctree: ../stubs/
-
-   CommutationChecker
```

Comment:
Isn't the `CommutationChecker` part of the public API we probably shouldn't remove this. (we had this exact problem as part of 1.1 or something where somebody removed it from the docs)